dev/com.ibm.ws.microprofile.metrics/resources/com/ibm/ws/microprofile/metrics/resources/Metrics.nlsprops

```diff
@@ -104,6 +104,27 @@ garbageCollectionCount.description=Displays the total number of collections that
 #Garbage collection time message
 garbageCollectionTime.description=Displays the approximate accumulated collection elapsed time in milliseconds. This attribute displays -1 if the collection elapsed time is undefined for this collector. The Java virtual machine implementation may use a high resolution timer to measure the elapsed time. This attribute may display the same value even if the collection count has been incremented if the collection elapsed time is very short.
 
+#-----------------------------------------------------------------------------------------------------------------------------
+# Monitor Metrics messages
+#-----------------------------------------------------------------------------------------------------------------------------
+threadpool.activeThreads.description=The number of threads that are actively executing tasks.
+threadpool.size.description=The size of the thread pool.
+
+servlet.request.total.description=The number of visits to this servlet since the start of the server.
+servlet.responseTime.total.description=The total response time of this servlet since the start of the server.
+
+connectionpool.create.total.description=The total number of managed connections that have been created since the pool creation.
+connectionpool.destroy.total.description=The total number of managed connections that have been destroyed since the pool creation.
+connectionpool.managedConnections.description=The current sum of managed connections in the free, shared, and unshared pools.
+connectionpool.connectionHandles.description=The number of connections that are in use. This number might include multiple connections that are shared from a single managed connection.
+connectionpool.freeConnections.description=The number of managed connections in the free pool.
+
+session.create.total.description=The number of sessions that have logged in since this metric is enabled.
```

Comment:
Change "since this metric **is** enabled" to "since this metric **was** enabled".